test/src/test/java/org/corfudb/integration/CorfuReplicationClusterConfigIT.java

```diff
@@ -1007,6 +1011,140 @@ public void testActiveLockRelease() throws Exception {
         assertThat(mapStandby.size()).isEqualTo(secondBatch);
     }
 
+    /**
+     * This test verifies log entry works after a force snap sync
+     * in the backup/restore workflow.
+     * <p>
+     * 1. Init with corfu 9000 active, 9001 standby, 9002 backup
+     * 2. Write 10 entries to active map
+     * 3. Start log replication: Node 9010 - active, Node 9020 - standby, Node 9030 - backup
+     * 4. Wait for Snapshot Sync, both maps have size 10
+     * 5. Write 5 more entries to active map, to verify Log Entry Sync
+     * 6. Change the topology, backup 9030 will be the new active
+     * 7. Trigger a force snapshot sync
+     * 8. Verify the force snapshot sync is completed
+     * 9. Write 5 entries to backup map, to verify Log Entry Sync
+     */
+    @Test
+    public void testBackupRestoreWorkflow() throws Exception {
+        Process backupCorfu = runServer(backupClusterCorfuPort, true);
+        Process backupReplicationServer = runReplicationServer(backupReplicationServerPort, nettyPluginPath);
+
+        CorfuRuntime.CorfuRuntimeParameters params = CorfuRuntime.CorfuRuntimeParameters
+                .builder()
+                .build();
+
+        CorfuRuntime backupRuntime = CorfuRuntime.fromParameters(params).setTransactionLogging(true);
+        backupRuntime.parseConfigurationString(backupCorfuEndpoint).connect();
+
+        CorfuTable<String, Integer> mapBackup = backupRuntime.getObjectsView()
+                .build()
+                .setStreamName(streamName)
+                .setTypeToken(new TypeToken<CorfuTable<String, Integer>>() {
+                })
+                .open();
+
+        assertThat(mapBackup.size()).isZero();
+
+        // Write 10 entries to active map
```

Comment:
This is nice, can you add a comment. Like "Initializing corfu servers with dummy data, to confirm data is wiped on startup" cause that is the purpose right?

Reply:
You mean dummy data on the standby side, right? at line 1059.
And it is used to make sure the standby corfu log tail is much longer than the backup one.